### DISCUSSION TITLE
Is it possible to do parent:state > child selectors?

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5945] [Q&A]
Probably not the best title... this is what I mean.

Suppose I have a card with an image and some content inside (a title and a paragraph)

```
<Card>
<img />
<h6>Cart title</h6>
<p>Card intro...</p>
</Card>
```

and I want to apply some effect to the image when the card is hovered

in regular CSS i would just write:
`.card:hover > img{...}`

Is there a way to do this kind of stuff with GrapesJs?

**Comment by artf** [comment]
At the low level, you can create your rule and pass it to the selector
```js
const rule = editor.Css.addRules(`.card:hover img { ... }`)[0];
editor.Selectors.select(rule);
```
This will allow you to edit the specific rule on the Style Manager level
